### PR TITLE
#160882082 Add flake8 for style checks

### PR DESCRIPTION
**Pull request by C3real-kill3r**
## What does this PR do?
It checks that the codebase is compliant with the pep8 style convention.

## Why is this important?
Currently, the codebase isn't pep8 compliant. This should be fixed by adding flake8 and ensuring that all flake8 checks pass.

#What are the relevant PT stories?
[160882082](https://www.pivotaltracker.com/story/show/160882082)

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[flake8]
+exclude = .venv, migrations, venv
+max-line-length = 125
```
Comment (esirK) [inline .flake8]: leave this at 119

**Comment by esirK** [comment]
This is what I get when I run `flake8`
Kindly add a screenshot of your result after running `flake8`
screen shot 2018-10-02 at 04 02 54

**Comment by C3real-kill3r** [comment]
> This is what I get when I run `flake8`
> Kindly add a screenshot of your result after running `flake8`
> screen shot 2018-10-02 at 04 02 54

I used sublime text to do the refactoring after installing the flake8 pluggin in it. Everything checked out. I did that because on the terminal I would be prompted to constantly run the flake8 over and over after making changes to ascertain that the codebase is compliant.

**Comment by esirK** [comment]
Kindly fix all the above errors.

**Comment by C3real-kill3r** [comment]
screen shot 2018-10-02 at 10 34 51
Everything is fixed now...and I've reduced the max-length to 119

**Comment by C3real-kill3r** [comment]
> screen shot 2018-10-02 at 10 34 51
> 
> Everything is fixed now...and I've reduced the max-length to 119

I've tested each file independently.

**Comment by esirK** [comment]
This doesn't run on my machine. Look for me at my workspace.

**Pull request merged**